fearless/Common/Model/KeystoreTag.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+
+enum KeystoreTag: String, CaseIterable {
+    case pincode
+
+    static func secretKeyTagForAddress(_ address: String) -> String { address + "-" + "secretKey" }
+    static func entropyTagForAddress(_ address: String) -> String { address + "-" + "entropy" }
+    static func deriviationTagForAddress(_ address: String) -> String { address + "-" + "deriv" }
+    static func seedTagForAddress(_ address: String) -> String { address + "-" + "seed" }
+}
+
+enum KeystoreTagV2: String, CaseIterable {
+    case pincode
+
+    static func substrateSecretKeyTagForMetaId(
+        _ metaId: String,
+        accountId: AccountId? = nil
+    ) -> String {
+        let suffix = "-substrateSecretKey"
+
+        return accountId.map { metaId + $0.toHex() + suffix } ?? metaId + suffix
```

Comment:
Although it is not necessary, I'd suggest to put this line into separate function 
`func bildTag(prefix: String = "", body: String?, suffix: String = "") -> String {...}` and call it like:
`buildTag(prefix: metaId, body: accountId.toHex(), suffix: "-substrateSecretKey")`

It would make code more readable and less error-prone